rust/Cargo.toml

```diff
@@ -19,5 +19,5 @@ opt-level = 3
 debug = false
 
 [patch.crates-io]
-zcash_note_encryption = { git = "https://github.com/zingolabs/librustzcash", rev = "acfdabc3629c93517e3c4742848996c595de6d3c" }
-zcash_primitives = { git = "https://github.com/zingolabs/librustzcash", rev = "acfdabc3629c93517e3c4742848996c595de6d3c" }
+zcash_note_encryption = { git = "https://github.com/zingolabs/librustzcash", rev = "fa67d9a645da55265ebea8cedbb77ff42317c726" }
```

Comment:
Can we match the cargo toml to zingolib so dependencies are the same
https://github.com/zingolabs/zingolib/blob/dev/Cargo.toml

Reply:
the commit `fa67d9a645da55265ebea8cedbb77ff42317c726` is the same in both, right?

Reply:
I removed them.